internal/tss/channels/webhook_channel.go

```diff
@@ -0,0 +1,77 @@
+package channels
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/alitto/pond"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/wallet-backend/internal/tss"
+	tssutils "github.com/stellar/wallet-backend/internal/tss/utils"
+	"github.com/stellar/wallet-backend/internal/utils"
+)
+
+type WebhookChannelConfigs struct {
+	HTTPClient           utils.HTTPClient
+	MaxBufferSize        int
+	MaxWorkers           int
+	MaxRetries           int
+	MinWaitBtwnRetriesMS int
+}
+
+type webhookPool struct {
+	Pool                 *pond.WorkerPool
+	HTTPClient           utils.HTTPClient
+	MaxRetries           int
+	MinWaitBtwnRetriesMS int
+}
+
+var WebhookChannelName = "WebhookChannel"
+
+var _ tss.Channel = (*webhookPool)(nil)
+
+func NewWebhookChannel(cfg WebhookChannelConfigs) *webhookPool {
+	pool := pond.New(cfg.MaxBufferSize, cfg.MaxWorkers, pond.Strategy(pond.Balanced()))
+	return &webhookPool{
+		Pool:                 pool,
+		HTTPClient:           cfg.HTTPClient,
+		MaxRetries:           cfg.MaxRetries,
+		MinWaitBtwnRetriesMS: cfg.MinWaitBtwnRetriesMS,
+	}
+
+}
+
+func (p *webhookPool) Send(payload tss.Payload) {
+	p.Pool.Submit(func() {
+		p.Receive(payload)
+	})
+}
+
+func (p *webhookPool) Receive(payload tss.Payload) {
```

Comment:
Can we add a test to validate this method errors? I see that you have a test that covers the "happy path", just wondering here if it's nice do validate these errors too